gtk4/src/auto/widget.rs

```diff
@@ -372,10 +372,6 @@ pub trait WidgetExt: 'static {
     #[doc(alias = "get_style_context")]
     fn style_context(&self) -> StyleContext;
 
-    #[doc(alias = "gtk_widget_get_template_child")]
-    #[doc(alias = "get_template_child")]
-    fn template_child(&self, widget_type: glib::types::Type, name: &str) -> glib::Object;
```

Comment:
Doesn't hurt to have it, does it?

Reply:
There is no point in having it, you can retrieve child widgets with `#[template_child]`. This is probably a helper for languages like Python